.golangci.yml

```diff
@@ -76,7 +76,8 @@ issues:
         - gocritic
         - ineffassign
         - govet
-    - path: main.go|service
+        - typecheck
+    - path: main.go|service|handlers
```

Comment:
Not sure about this - excluding typecheck doesn't follow the standards. This linter version is also quite old